### PR TITLE
change: updated resty-radixtree to v1.5.

### PR DESCRIPTION
**Pull request by membphis**

### CODE REVIEW

File: doc/admin-api-cn.md [hunk 1]
```diff
@@ -32,7 +32,8 @@
...
 |remote_addrs|可选 |匹配规则|列表形态的 `remote_addr`，表示允许有多个不同 IP 地址，符合其中任意一个即可。|{"127.0.0.1", "192.0.0.0/8", "::1"}|
 |methods  |可选 |匹配规则|如果为空或没有该选项，代表没有任何 `method` 限制，也可以是一个或多个的组合：`GET`, `POST`, `PUT`, `DELETE`, `PATCH`, `HEAD`, `OPTIONS`，`CONNECT`，`TRACE`。|{"GET", "POST"}|
-|vars       |可选  |匹配规则(仅支持 `radixtree` 路由)|由一个或多个`{var, operator, val}`元素组成的列表，类似这样：`{{var, operator, val}, {var, operator, val}, ...}`。例如：`{"arg_name", "==", "json"}`，表示当前请求参数 `name` 是 `json`。这里的 `var` 与 NGINX 内部自身变量命名是保持一致，所以也可以使用 `request_uri`、`host` 等；对于 `operator` 部分，目前已支持的运算符有 `==`、`~=`、`>`和`<`，特别的对于后面两个运算符，会把结果先转换成 number 然后再做比较。|{{"arg_name", "==", "json"}, {"arg_age", ">", 18}}|
+|vars       |可选  |匹配规则(仅支持 `radixtree` 路由)|由一个或多个`{var, operator, val}`元素组成的列表，类似这样：`{{var, operator, val}, {var, operator, val}, ...}`。例如：`{"arg_name", "==", "json"}`，表示当前请求参数 `name` 是 `json`。这里的 `var` 与 NGINX 内部自身变量命名是保持一致，所以也可以使用 `request_uri`、`host` 等；对于 `operator` 部分，目前已支持的运算符有 `==`、`~=`、`>`、`<` 和 `~~`。对于`>`和`<`两个运算符，会把结果先转换成 number 然后再做比较。查看支持的[运算符列表](#运算符列表)|{{"arg_name", "==", "json"}, {"arg_age", ">", 18}}|
```
Comment (moonming) [inline doc/admin-api-cn.md]: please add the English version at the same time.

Comment (moonming) [inline doc/admin-api-cn.md]: `Nginx`, not `NGINX.`

File: doc/admin-api-cn.md [hunk 2]
```diff
@@ -32,7 +32,8 @@
...
+|filter_fun|可选|匹配规则|用户自定义的过滤函数。可以使用它来实现特殊场景的匹配要求实现。该函数默认接受一个名为 vars 的输入参数，可以用它来获取 Nginx 变量。|function(vars) return vars["arg_name"] == "json" end|
```
Comment (moonming) [inline doc/admin-api-cn.md]: `filter_func` is more clear.

Comment (moonming) [reply doc/admin-api-cn.md]: `可以使用它来实现自定义的路由匹配规则`  instead of `可以使用它来实现特殊场景的匹配要求实现`

File: doc/admin-api-cn.md [hunk 3]
```diff
@@ -67,5 +68,35 @@ Date: Sat, 31 Aug 2019 01:17:15 GMT
 
 目前是直接返回与 etcd 交互后的结果。
 
+### 运算符列表
+
+|运算符   |描述       |例子|
+|--------|-----------|-------|
+|==      |相等      |{"arg_name", "==", "json"}|
```
Comment (moonming) [inline doc/admin-api-cn.md]: only for string compare?

File: doc/admin-api-cn.md [hunk 4]
```diff
@@ -67,5 +68,35 @@ Date: Sat, 31 Aug 2019 01:17:15 GMT
...
+|~=      |不等于    |{"arg_name", "~=", "json"}|
+|>       |大于      |{"arg_age", ">", 24}|
```
Comment (moonming) [inline doc/admin-api-cn.md]: only for number compare?

File: doc/admin-api-cn.md (hunk 1 above)
Comment (membphis) [reply doc/admin-api-cn.md]: Please note that this document does not have an English version.

File: doc/admin-api-cn.md (hunk 3 above)
Comment (membphis) [reply doc/admin-api-cn.md]: Yes, it is a string object if we get value from 'ngx.var'.

File: doc/admin-api-cn.md (hunk 4 above)
Comment (membphis) [reply doc/admin-api-cn.md]: Please view this doc fully, I have wrote this case.

`对于`>`和`<`两个运算符，会把结果先转换成 number 然后再做比较。`

File: doc/admin-api-cn.md (hunk 2 above)
Comment (membphis) [reply doc/admin-api-cn.md]: Good catch

**Comment by membphis** [comment]
@moonming please take a look

**Pull request merged**